pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[project]
+name = "lundpython"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "astropy>=7.0.0",
+    "jupyter>=1.1.1",
+    "jupyterlab-rise>=0.42.0",
+    "matplotlib>=3.9.2",
+    "numpy>=2.1.3",
+]
```

Comment:
Is there a good reason for not complying with [SPEC 0](https://scientific-python.org/specs/spec-0000/)?

Reply:
What part of this is in violation of SPEC 0?
It seems to me the spec is about not supporting or requiring old versions of packages, and all these are new.

Reply:
Specifying how long specific versions of dependencies should be supported is the entire point of the SPEC:

> Specifically, we recommend that:
>
>   1. Support for Python versions be dropped 3 years after their initial release.
>   2. Support for core package dependencies be dropped 2 years after their initial release.

Although here it makes sense to support versions that will be 2 years old next autumn instead of right now.